app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,26 @@
+.banner {
+  background-color: $orange;
+  display: block;
+  height: 30px;
+  width: auto;
+}
+
+.main-section {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 40px;
+
+  .gradient-line {
+    background: linear-gradient(0deg, $white, $orange 40%, $white);
+    height: 400px;
+    margin: 0 50px;
+    width: 5px;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .phone,
+  .gradient-line { display: none;}
```

Comment:
although this is not wrong, we tend to always put properties in a separate line, like this:
```
.phone,
.gradient-line {
  display: none;
}